### ISSUE TITLE
foreach format

### ISSUE THREAD

**Issue by sergeyklay** [cheshirekow/cmake_format#147]
Hello,

Is there any way to format [`foreach`](https://cmake.org/cmake/help/v3.16/command/foreach.html) statement more nicely?

**Expected**
```cmake
foreach(item IN LISTS ARGN)
# ...
endforeach()
```

**Actual**
```cmake
foreach(item
        IN
        LISTS
        ARGN)
# ...
endforeach()
```

**Comment by cheshirekow** [comment]
I wasn't aware of the `IN LISTS` syntax. I don't think I have parse logic for it. We can add it easily enough though!

**Comment by cheshirekow** [comment]
Fixed in `v0.6.2`.
